### PR TITLE
Add link to Kubernetes Release Calendar for burndown meetings

### PR DESCRIPTION
**Pull request by dims**

### CODE REVIEW

**Comment by dims** [comment]
@justaugustus can you please explain the `/hold`? does this change not reflect current status?

File: releases/release-1.16/README.md [hunk 1]
```diff
@@ -88,7 +88,7 @@ point we have a list of outstanding issues and PRs, and are actively working to
 burn that list down.  SIG representatives will be asked to attend if their SIG has
 specific outstanding issues that are blocking the release.
 
-Join [kubernetes-sig-release@] to get a calendar invite.  This meeting
+Please subscribe to [Kubernetes Release Calendar] for joining this Meeting, it
```
Comment (justaugustus) [inline releases/release-1.16/README.md]:
```suggestion
Please subscribe to the [Kubernetes Release Calendar][kubernetes-release-calendar] to see meeting events. Burndown meetings
```

**Comment by justaugustus** [comment]
Thanks for adding a note to clarify this, @dims!
/lgtm
/approve
/hold cancel

**Pull request merged**